playbooks/roles/ocp-install/tasks/checkcert.yaml

```diff
@@ -0,0 +1,31 @@
+---
+# tasks file for ocp4 install cert
+
+- name: check if oc is installed
+  command: type -P oc
+  register: oc_installed
+
+# if cluster is more than  24 hrs. old set over24 to 1
+- name: check the cluster timestamp
+  shell: |
+    # get the cluster create time in seconds
+    tmstmp=`date -d$(oc get -o json clusterversion version |  jq -r '.metadata.creationTimestamp') '+%s'`
+    echo $(( $tmstmp  + 86400 < `date +%s` ? 1 : 0 ))
+  register: over24
+  when: oc_installed.rc == 0
+
+# update worker.ign file with new cert on HTTP Server if cluster age > 24 hrs(under24=0).
```

Comment:
We can add a link to this as comment - https://access.redhat.com/solutions/4799921
Also the var is over24 so the comment above `under24=0` should change to `over24==1`

Reply:
Hi Pradipta,
I have made changes to look at the certificate validity dates to see if we have valid or expired certificates.
First run it looks good. Please review.

Reply:
Definitely, I will add a link to the comment.